package.json

```diff
@@ -72,7 +72,7 @@
     "express": "^4.18.2",
     "express-session": "^1.17.3",
     "fs-extra": "^11.1.0",
-    "govuk-frontend": "^4.5.0",
+    "govuk-frontend": "github:alphagov/govuk-frontend#6d1a4ef0a",
```

Comment:
Until GOV.UK Frontend v5 is released I've used a preview branch for testing

### Legacy colour palette
The legacy palette was replaced in GOV.UK Frontend v3.0.0:
https://designnotes.blog.gov.uk/2019/07/29/weve-updated-the-gov-uk-colours-and-font/

Some calls to `govuk-colour()` in this repo [and `@govuk-prototype-kit/step-by-step`](https://github.com/search?q=repo%3Aalphagov%2Fgovuk-prototype-kit-step-by-step+%22%24legacy%22&type=code) still include a `$legacy` palette fallback colour so not sure it's still needed?

We'll need other PRs to either remove it or suppress the warning:

```console
WARNING: The `$legacy` parameter of `govuk-colour` is deprecated and is non-operational. It will be removed in the next major version. To silence this warning, update $govuk-suppressed-warnings with key: "legacy-colour-param"
    node_modules/govuk-frontend/dist/govuk/helpers/_colour.scss 25:5                                govuk-colour()
    node_modules/@govuk-prototype-kit/step-by-step/sass/_step-by-step-navigation-header.scss 10:15  @import
    .tmp/sass/_plugins.scss 9:9                                                                     @import
    node_modules/govuk-prototype-kit/lib/assets/sass/prototype.scss 5:9                             @import
    node_modules/govuk-prototype-kit/lib/assets/sass/unbranded.scss 12:9                            root stylesheet
```

Reply:
I've temporarily written this line to your Sass variables:

```scss
$govuk-suppressed-warnings: (legacy-colour-param);
```